### ISSUE TITLE
train_dataset is also the test-dataset? 

### ISSUE THREAD

**Issue by guotong1988** [ZJU-DAILY/SGDL#2]
https://github.com/ZJU-DAILY/SGDL/blob/main/code/main.py

<img width="609" alt="image" src="https://user-images.githubusercontent.com/4702353/165045959-69c59077-549e-45ce-b182-bd57ec4e9d2c.png">

I am new to denoise. Thank you very much! @Scottdyt

**Comment by RacheAndolini** [comment]
Thanks for the question.
The trainning set, validation set and test set are all integrated in the Loader module(i.e. train_dataset in main.py). Please check the file dataloader.py for detailed information. 
![image](https://user-images.githubusercontent.com/59520565/165049942-bed5a600-5361-4bf0-84d0-053d976e22ab.png)
